common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -662,6 +688,32 @@ def get_problem_html(self, encapsulate=True):
 
         return html
 
+    def remove_extra_tags(self, html):
+        """
+        The capa xml includes many tags such as <additional_answer> or <demandhint> which are not
+        meant to be part of the client html. We strip them all and return the resulting html.
+        """
+        tags = ['demandhint', 'choicehint', 'optionhint', 'stringhint', 'numerichint', 'optionhint',
```

Comment:
My only reason for wanting it to be a constant is it'll be really easy for someone to overlook adding to this list if they add a new problem type in - but I guess looking at it a second time, these aren't problem types but very specific hint-related capa, so you'd only need to add to this if you were working on hints... right?

If that's the case, I would recommend updating the docstring to indicate how this method is specific to the hinting infrastructure.